content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown

```diff
@@ -0,0 +1,72 @@
+---
+title: Mostrar la fecha actual automáticamente en un informe Rmarkdown
+author: Thomas Neitmann
+date: '2020-03-22'
+slug: rmarkdown-current-date
+categories:
+  - bytesized
+  - R
+tags:
+  - rmarkdown
+  - knitr
+toc: no
+images: ~
+---
+
+```{r, include=FALSE}
+Sys.setenv(lang = "en_US")
+knitr::opts_chunk$set(collapse = TRUE)
+```
+
+<!-- `Rmarkdown` is a great tool for creating data-driven reports that have to be updated on a regular basis. To communicate to stakeholders that the report is indeed up to date it's important to display the current date below the title. -->
+
+`Rmarkdown` es una herramienta fantástica para crear informes impulsados por datos que requieran actualizarse regularmente. Para que los lectores sepan que el informe está realmente actualizado, es importante mostrar la fecha actual debajo del título.
+
+<!-- An obvious solution to display the current date is to update the date field in the YAML header manually before knitting. But this is both error prone and you are likely to forget about it. Instead you can dislay the current date in an automatic way using the method below. -->
+
+Una solución obvia para mostrar la fecha actual sería modificar manualmente el campo de la fecha en el encabezado YAML antes de compilar el informe (**knit**). Pero esto puede conllevar errores y es probable que se te olvide actualizarlo. En cambio, puedes mostrar la fecha actualizada automáticamente usando el método a continuación.
+
+```{r, eval=FALSE}
+---
+title: "Super Important Report"
```

Comment:
Could you translate the `title` as well?

Reply:
Solved